CoastalResilienceNetwork/GeositeFramework
# Add Plugin "Minimize" button

## Overview

This PR adds a minimize button to the sidebar plugin header. The feature had been available in the prior version, but until now we hadn't yet implemented it in the prototype. Here we add an underscore icon to the sidebar plugin header, and call the `.deselect()` method when it's clicked. This removes the plugin from the map viewport but leaves its layers active.

### Demo

![screen shot 2016-12-20 at 10 43 02 am](https://cloud.githubusercontent.com/assets/4165523/21357077/ebabad06-c6a1-11e6-8c46-e8008556a71f.png)

& ->

![screen shot 2016-12-20 at 10 43 12 am](https://cloud.githubusercontent.com/assets/4165523/21357083/f0d6d1e8-c6a1-11e6-8b4e-357bd1da6233.png)

### Notes

The code to add the minimize button includes an `<i class="icon-minimize">` element which is used primarily to size the button properly, in accordance with the other sidebar header controls.

The first and third commits here involve some cleanup work:
- in d0ed543, we remove the `map-utils` plugin references from the csproj file. The map-utils are now part of the core framework -- and not a plugin -- but the references hadn't been removed from Visual Studio, which meant the plugin dir had to be deleted manually to get the project to build
- in 4af0cb5 I cut a reference to `plugin-eye`, which was added in #702 then subsequently removed

## Testing Instructions

 * get and build the project and open it in the browser, along with the dev tools
 * open the Regional Planning plugin, pan to New York, and turn on the Hurricane Sandy layers
 * click the "_" button to minimize and verify that the sidebar plugin disappears but "Regional Planning" remains highlighted and the layers remain on the map

Connects #785
